lib/broccoli/ember-app.js

```diff
@@ -1599,7 +1600,7 @@ class EmberApp {
         overwrite: true,
       }));
 
-      if (options.minifyCSS.enabled === true) {
+      if (cssMinificationEnabled === true) {
         preprocessedStyles = preprocessMinifyCss(preprocessedStyles, options);
```

Comment:
Don't you want to refactor `preprocessMinifyCss` to expect `options.minifyCSS` up a level? Or is this more of a temp fix?